package.json

```diff
@@ -36,6 +36,7 @@
     "build": "node scripts/babelcheck.js && npm run build:res && npm run build:bundle",
     "build:dev": "node scripts/babelcheck.js && npm run build:res && npm run build:bundle:dev",
     "dist": "scripts/package.sh",
+    "postinstall": "cd electron && npm i",
```

Comment:
Currently `npm i` is only useful because it triggers prepublish, in a later version of npm this functionality will no longer work as prepublish will only be hit on `npm publish`.  I guess the beter alternative to installing the electron deps here is to define scripts for electron-builder which run npm i on the electron deps first.